micro-osd.sh

```diff
@@ -77,7 +77,7 @@ ceph-mon --id ${MON_NAME}
 # start an osd
 OSD_ID=$(ceph osd create)
 ceph osd crush add osd.${OSD_ID} 1 root=default host=localhost
-ceph-osd --id ${OSD_ID} --mkjournal --mkfs
+ceph-osd --id ${OSD_ID} --mkjournal --mkfs -f
```

Comment:
I prefer the self-describing `--long-named-options`, as it is not clean if `-f` stands for `--force` or something else. Is there long form option?

Reply:
+1 to long options in scripts. In this case '-f' means '--foreground' according to https://docs.ceph.com/en/latest/man/8/ceph-osd/  which makes sense given the PR title.

Reply:
According to `ceph-osd -h` there is no long version, else I would have used it, but maybe that output is wrong?

Reply:
Ok, despitr the command help output `--foreground` seems to work. However, after reading the better description in the man page, I doubt this has an effect. So I also added a sync. Let's see, if it works. If not we can go with a sleep.